Hymns/Home/HomeView.swift

```diff
@@ -23,22 +23,34 @@ struct HomeView: View {
                 .padding(.horizontal)
                 .padding(.top, viewModel.searchActive ? nil : .zero)
 
-            viewModel.label.map {
-                Text($0).fontWeight(.bold).padding(.top).padding(.leading).foregroundColor(Color("darkModeSubtitle"))
+            if !viewModel.songResults.isEmpty {
+                viewModel.label.map {
+                    Text($0).fontWeight(.bold).padding(.top).padding(.leading).foregroundColor(Color("darkModeSubtitle"))
+                }
```

Comment:
I don't mean to be nitpicky, but I think this is better done in the viewmodel, than the view. It's business logic (as opposed to view logic) and can be unit tested if it lived in the viewmodel instead

Reply:
Since it's technically outside the scope of this PR we can leave a TODO and do this label hiding later as well

Reply:
Done, just going to leave the TODO.